cinema/views.py

```diff
@@ -56,3 +99,30 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderPagination(PageNumberPagination):
+    page_size = 2
+    page_size_query_param = "page_size"
+    max_page_size = 100
+
+
+class OrderViewSet(viewsets.ModelViewSet):
+    pagination_class = OrderPagination
+
+    def get_queryset(self) -> QuerySet:
+        queryset = Order.objects.prefetch_related(
+            "tickets__movie_session",
+            "tickets__movie_session__cinema_hall",
+            "tickets__movie_session__movie",
+        )
+        return queryset
+
+    def get_serializer_class(self):
+        if self.action == "list":
+            return OrderListSerializer
+        if self.action == "create":
+            return OrderSerializer
+
+    def perform_create(self, serializer):
+        serializer.save(user=self.request.user)
```

Comment:
![image](https://user-images.githubusercontent.com/105493988/231223574-2107d795-cd48-4c3c-9e0b-342b5fdeff57.png)

Reply:
> ![image](https://user-images.githubusercontent.com/105493988/231223574-2107d795-cd48-4c3c-9e0b-342b5fdeff57.png)

Sorry, but image is broken